rubin_scheduler/scheduler/features/conditions.py

```diff
@@ -310,17 +314,14 @@ def _init_attributes(self):
 
         # Telescope limits - these can be None
         # These should be in radians.
-        self.tel_az_limits = None
-        self.tel_alt_limits = None
+        self.sky_az_limits = None
+        self.sky_alt_limits = None
         # Kinematic model (real slew) limits - these can't be None
         # if you are using the AltAzShadowMaskBasisFunction.
-        # These generous limits won't restrict the AltAzShadowMask.
-        # Radians.
-        self.kinematic_alt_limits = [np.radians(-10), np.radians(100)]
-        self.kinematic_az_limits = [np.radians(-250), np.radians(250)]
-        # This has a (reasonable) default value, to avoid failure of
-        # AltAzShadowMask in case this isn't set otherwise.
-        self.altaz_limit_pad = np.radians(2)
+        # Setting default values here that will not restrict any choices,
+        # but just avoiding "None".  Values in Radians.
+        self.tel_alt_limits = [np.radians(-10), np.radians(100)]
+        self.tel_az_limits = [np.radians(-270), np.radians(270)]
```

Comment:
I was just trying to avoid an extra if downstream, but I'm ok with setting the defaults to None instead of non-blocking values. 
I do think it's possible that we should have *more* defaults in the Conditions, rather than Nones. It depends though on if we want to have default values for missing telemetry set outside the FBS (at the SchedulerCSC), or inside.